src/test/java/com/parkit/parkingsystem/integration/ParkingDataBaseIT.java

```diff
@@ -37,29 +57,59 @@ private static void setUp() throws Exception{
 
     @BeforeEach
     private void setUpPerTest() throws Exception {
-        when(inputReaderUtil.readSelection()).thenReturn(1);
+        when(inputReaderUtil.readSelection()).thenReturn(2);
         when(inputReaderUtil.readVehicleRegistrationNumber()).thenReturn("ABCDEF");
-        dataBasePrepareService.clearDataBaseEntries();
+       // dataBasePrepareService.clearDataBaseEntries();
```

Comment:
Dead code

Reply:
Can't be deleted right away, will include it in @Before of test for exiting vehicle